api/recover.go

```diff
@@ -32,12 +32,6 @@ func (a *API) Recover(w http.ResponseWriter, r *http.Request) error {
 
 	aud := a.requestAud(ctx, r)
 	user, err := models.FindUserByEmailAndAudience(a.db, instanceID, params.Email, aud)
-	if err != nil {
```

Comment:
i think we still need to check if it's an error here and return some generic error message? otherwise it will panic [here](https://github.com/supabase/gotrue/pull/359/files#diff-fe4851c1c04b9250f3e6f83565798c83a3b1e65a7af256711ff89fab7cae627cR37) if the user is nil

Reply:
Thanks for the catch! That makes sense. I'm not too sure we can return an error message though since any error message would give away information (since GoTrue is OSS and the attacker can look up the code). Might make sense to return 200/ Will think about this and address later in the evening.